lib/menu.rb

```diff
@@ -15,5 +16,29 @@ def list_of_dishes
 
   def print_menu
     list_of_dishes
+=======
+class Menu
+
+  # create attr_reader to pass method
+  attr_reader :list_of_dishes
+
+  def initialize(list_of_dishes)
+    @list_of_dishes = list_of_dishes
```

Comment:
Can add the menu inside as an instance variable of Menu instead of inputting it each time